.env.example

```
@@ -36,6 +40,61 @@ NOTION_WEBHOOK_SECRET=
+CMS_WEBHOOK_EVENTS_RETENTION_DAYS=30
+CMS_WEBHOOK_EVENTS_RETENTION_LIMIT=100
+# Optional Image Jobs cleanup retention (non-winner completed rows only).
+CMS_IMAGE_JOBS_RETENTION_DAYS=30
+CMS_IMAGE_JOBS_CLEANUP_LIMIT=100
+# Optional safety valve: when true, allows age-based fallback cleanup using last edited time
+# if a row has no explicit Retention Until/Cleanup After date.
+CMS_IMAGE_JOBS_CLEANUP_ALLOW_FALLBACK_AGE=false
+
+# Content calendar idea seeding cron.
+CALENDAR_SEEDING_ENABLED=true
+CALENDAR_SEEDING_DRY_RUN=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CALENDAR_SEEDING_DRY_RUN key should go before the CALENDAR_SEEDING_ENABLED key

(UnorderedKey)